examples/intro_to_inverse_problems.jl

```diff
@@ -0,0 +1,219 @@
+# # Intro to `InverseProblem`
+#
+# This example illustrates the construction of an "ensemble simulation" that can
+# evaluate an ensemble of column models given an ensemble of free parameter sets.
+# The example then builds an `InverseProblem` from observations, an ensemble simulation,
+# and a set of free parameters, and illustrates its basic usage.
+#
+# ## Install dependencies
+#
+# First let's make sure we have all required packages installed.
+
+# ```julia
+# using Pkg
+# pkg"add OceanTurbulenceParameterEstimation, Oceananigans, Distributions, CairoMakie"
+# ```
+
+# First we load few things
+
+using OceanTurbulenceParameterEstimation
+
+using Oceananigans
+using Oceananigans.Units
+using Oceananigans.Models.HydrostaticFreeSurfaceModels: ColumnEnsembleSize
+using Oceananigans.TurbulenceClosures: ConvectiveAdjustmentVerticalDiffusivity
+
+using CairoMakie
+using Distributions
+using JLD2
+
+# We reuse some utilities from a previous example to build observations:
+
+examples_path = joinpath(pathof(OceanTurbulenceParameterEstimation), "..", "..", "examples")
+include(joinpath(examples_path, "intro_to_observations.jl"))
```

Comment:
So what does this line mean to do?
This will redraw all figures in the `intro_to_observations.jl`, no? That's what you wanna do here?

I feel you only want to have the definition of certain functions available, right?